kaggle-classification/keras_trainer/model.py

```diff
@@ -182,13 +187,25 @@ def _setup_embeddings_matrix(self):
   parser.add_argument(
       '--batch_size', type=int, default=64, help='Batch size.')
 
+
   FLAGS = parser.parse_args()
 
   hparams = DEFAULT_HPARAMS
   hparams.learning_rate = FLAGS.learning_rate
   hparams.dropout_rate = FLAGS.dropout_rate
   hparams.batch_size = FLAGS.batch_size
 
+  if FLAGS.comet_key:
+    experiment = Experiment(api_key=FLAGS.comet_key, 
+      project_name='comet_trial_run', 
```

Comment:
should this be flag? up to you

Reply:
Yes, we want this to work even if you don't have a comet API key.

Reply:
I was referring to project_name. But actually it sounds like we might all be sharing one project?